packages/core-bridge/index.d.ts

```diff
@@ -145,17 +148,48 @@ export interface TelemetryOptions {
 
   /**
    * Control where to send Rust Core logs
-   *
-   * @default log to console
    */
-  logging?: Logger;
+  logging?:
+    | {
+        /**
+         * A string in (env filter format)[https://docs.rs/tracing-subscriber/0.2.20/tracing_subscriber/struct.EnvFilter.html]
+         * which determines the verboseness of logging output.
+         *
+         * You can use {@link Runtime.makeTracingFilterString()} to easily build a correctly formatted filter
+         * string based on desired log level for Core SDK and other native packages.
+         *
+         * @default `makeTracingFilterString('INFO', 'INFO')`
+         */
+        filter?: string;
+
+        /**
+         * A string in (env filter format)[https://docs.rs/tracing-subscriber/0.2.20/tracing_subscriber/struct.EnvFilter.html]
+         * which determines the verboseness of logging output.
+         *
+         * You can use {@link Runtime.makeTracingFilterString()} to easily build a correctly formatted filter
+         * string based on desired log level for Core SDK and other native packages.
+         *
+         * @default `console`
+         */
+        mode?: 'console' | 'forward';
```

Comment:
I don't love that we're now inconsistent with `tracing` and `metrics`.
Here's we're using `mode` and in the other options, we're using a union type.

I'd make these consistent, probably preserving the structure we had previously even if it means we'll have empty options in `console` and `forward`.

Reply:
Reverted to `console: {} | forward: {}` instead of `mode: 'console' | 'forward'`